src/app/pages/large-display/large-display.component.ts

```diff
@@ -90,6 +92,7 @@ export class LargeDisplayComponent implements OnInit {
     this.teacherRoundService.endRound();
   }
 
+  @HostListener('window:keyup.Space')
```

Comment:
We're going to want to be careful with this. This will react to *all* spacebar events in the large display component and this probably includes when typing in a field. It is not unreasonable that we will have things like text fields in the large display and this will play/pause the round anytime the user hits space while typing. This is also likely an accessibility issue because users may want to navigate the app with their keyboard and use space to select things.

Also this is another reason we should split the "during round" stuff into its own component.